app/models/receipt.rb

```diff
@@ -0,0 +1,16 @@
+class Receipt < ActiveRecord::Base
+    belongs_to :user
```

Comment:
Receipts should belong to both a buyer and a seller, not just a buyer. This way seller can view their receipts.